pulpcore/cli/rpm/comps.py

```diff
@@ -0,0 +1,56 @@
+import gettext
+from typing import IO, Optional
+
+import click
+
+from pulpcore.cli.common.context import (
+    PluginRequirement,
+    PulpContext,
+    pass_entity_context,
+    pass_pulp_context,
+)
+from pulpcore.cli.rpm.context import PulpRpmCompsXmlContext, PulpRpmRepositoryContext
+
+_ = gettext.gettext
+
+
+@click.group()
+@click.option(
+    "-t",
+    "--type",
+    "repo_type",
+    type=click.Choice(["rpm"], case_sensitive=False),
+    default="rpm",
+)
```

Comment:
With the resource option, we don't do this anymore.

Reply:
I meant the whole "-t" option seems superfluous at this point. Also as you just explained, the "comps" is not a "thing" in Pulp. So i would be ok to say we can merge the command in to "`comps-upload`". WDYT?

Reply:
I think this all sounds like a fine idea. Ready for review.